apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -661,6 +688,9 @@ where
                 e
             )
         });
+        // TODO(namada#1041): we check the Ethereum oracle is started on every
```

Comment:
Why can we just do this check in the `Shell::new` method if block height > 0 and also call it in `init_chain`?

Reply:
Yeah this sounds like the best solution! I've updated issue https://github.com/anoma/namada/issues/1041 - will merge this PR and open a new PR to optimize this.

For Ethereum bridge parameters that may be allowed to be changed by governance vote (at the moment, that's just the `min_confirmations` parameter) - we will need to check at the beginning of each epoch if that's changed, will open a separate issue for that.

Reply:
We can do a check in `Shell::new` something like:

```
        // we may be able to start the Ethereum oracle right away if we have
        // already synced at least one block
        if shell.storage.last_height >= BlockHeight(1) {
            shell.ensure_ethereum_oracle_started()
        }
```